ExtensionFrSexEst.owl

```diff
@@ -385,380 +307,408 @@
     </owl:Class>
     
 
 
-    <!-- http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#SexScoreMylohyoidLine -->
+    <!-- http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#CranialAssay -->
 
-    <owl:Class rdf:about="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#SexScoreMylohyoidLine">
-        <rdfs:subClassOf rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#MandibularSexScore"/>
+    <owl:Class rdf:about="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#CranialAssay">
+        <rdfs:subClassOf rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#Assay.FrSexEst"/>
     </owl:Class>
     
 
 
-    <!-- http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#SexScoreNuchalPlanum -->
+    <!-- http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#DataTransformation.FrSexEst -->
 
-    <owl:Class rdf:about="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#SexScoreNuchalPlanum">
-        <rdfs:subClassOf rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#CranialSexScore"/>
+    <owl:Class rdf:about="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#DataTransformation.FrSexEst">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0200000"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/BFO_0000050"/>
+                <owl:someValuesFrom rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#StudyDesignExecution.FrSexEst"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/OBI_0000299"/>
+                <owl:someValuesFrom rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#DegreeOfSexualization"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/OBI_0000417"/>
+                <owl:hasValue rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#DataTransformationObjective.FrSexEst"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
     </owl:Class>
```

Comment:
How does the data transformation work? This needs to be specified by the input statements.

1. If the sex scores from each assay are to be transformed separately, you will need one data transformation (class) for each assay.
2. If the sex scores from all assays are to be transformed into one degree of sexualization, all of them will need to be specified as input.

Any way, you will need to reference sex scores by the assays that produced them. This can be done either with complex restriction statements (e.g. "'has specified output' some SexScore and 'is specified output of' some Assay.Glabella") or by creating subclasses of class FrSexEst:SexScore). As the complex restriction statements are unlikely to be evaluated by VITRO, I would go with the subclasses.